### PR TITLE
Enhancement: Run tests on GitHub Actions

### PR DESCRIPTION
**Pull request by localheinz**
This pull request

- [x] runs tests on GitHub Actions

### CODE REVIEW

File: .github/workflows/integrate.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+          extensions: "none, dom, json, mbstring, tokenizer, xml, xmlwriter"
+          php-version: "${{ matrix.php-version }}"
+
+      - name: "Set up problem matchers for PHP"
+        run: "echo \"::add-matcher::${{ runner.tool_cache }}/php.json\""
+
+      - name: "Run tests"
+        run: "php tests/run-tests.php"
+        env:
+          NO_INTERACTION: "true"
```
Comment (cmb69) [inline .github/workflows/integrate.yaml]: I think we should add some useful arguments. First `-q` is equivalent to setting `NO_INTERACTION`. Then, at least in the long run, parallel test execution makes sense (`-j2` or maybe `-j3` seem to be good for GH actions, currently). And `--show-diff` is helpful to diagnose failing tests.

We could also create a `junit.out.xml` by setting the environment variable `TEST_PHP_JUNIT` to the desired filename, and then packing the file for download. Might be overkill for now, though.

**Comment by cmb69** [comment]
Thanks for the PR!

File: .github/workflows/integrate.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+jobs:
+  tests:
+    name: "Tests"
+
+    runs-on: "ubuntu-18.04"
+
+    strategy:
+      matrix:
+        php-version:
+          - "7.2"
```
Comment (localheinz) [inline .github/workflows/integrate.yaml:20]: Using PHP 7.2 here as laid out in `README.md`.

Comment (cmb69) [reply .github/workflows/integrate.yaml:20]: Okay, let's go with PHP 7.2 then (although I'm not sure which version actually runs on php.net).

**Comment by localheinz** [comment]
Thank you, @cmb69!

**Pull request merged**